content/en/docs/reference/glossary/eviction.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Eviction
+id: eviction
+date: 2020-05-06
+full_link: /docs/tasks/administer-cluster/out-of-resource/
```

Comment:
When I see `full_link`s that point to task pages, a little imaginary alarm light comes on.

Reply:
I am not quite sure what is the use of the `full_link`. Honestly, I just used the glossary entry for Annotations as a template to create this entry.
Does the `full_link` need to point to a concept page? There is no concept page for eviction. The link that I have added is the first page that comes up when you do a site search for 'eviction'.